k2/csrc/ragged_ops.h

```diff
@@ -388,10 +400,17 @@ Ragged<T> Transpose(Ragged<T> &src,
                            same value
       @param [in] num_srcs Number of source shapes to append; require
                            num_srcs > 0.
+      @param [out] merge_map  If not nullptr, will be set to the merge-map
```

Comment:
it's better if we can say `meger_map.Dim() == ans.Dim()`,  even if user can figure this out with `0 <= i < ans.NumElements()`